### DISCUSSION TITLE
Hosted builder for easy creation

### DISCUSSION THREAD

**Discussion by Onkeyuk** [kudrykv/latex-yearly-planner#82] [Ideas]
Love this template but I’m not very technical for creating myself so I’ve been using prebuilt. 

It would be lovely to have a simple hosted area where we can pick some options like  template type, name, pick a cover image, extras like extra notes pages etc and then it generates

**Comment by fishzle** [comment]
that would cost money to run a server. I think if you want the options you might find someone on etsy who could customise for you. Maybe https://www.etsy.com/shop/EightVBStudio (I have relation to it, it was just first hit in goog)
